---------------------
PatchSet 18 
Date: 2010/08/17 00:33:57
Author: leon
Branch: HEAD
Tag: (none) 
Log:
More documentation for org.gk.gpml.

Members: 
	package.html:1.1->1.2 

Index: GKB/java/authorTool/src/org/gk/gpml/package.html
diff -u GKB/java/authorTool/src/org/gk/gpml/package.html:1.1 GKB/java/authorTool/src/org/gk/gpml/package.html:1.2
--- GKB/java/authorTool/src/org/gk/gpml/package.html:1.1	Fri Oct 23 19:12:50 2009
+++ GKB/java/authorTool/src/org/gk/gpml/package.html	Mon Aug 16 12:33:57 2010
@@ -3,6 +3,20 @@
 		<title>Description for Package org.gk.gpml</title>
 	</head>
 	<body>
-		This package is used to export Reactome pathways into GPML, the format used by wikipathways.
+		<p>This package is used to export Reactome pathways into GPML, the format used by wikipathways, and vice versa.</p>
+		<p>Note that due to the nature of GPML file itself, conversion back to Reactome pathways is flaky at best.
+		Currently the conversion code can only convert back GPML files that are produced from ReactomeToGPMLConverter.</p>
+		
+		<p>Executable Java classes (with main(String[]) method) usage documentation is included in each of the source codes.</p>
+		
+		<p>Note that classes that do conversion from GPML to Reactome needs Pathvisio classes to run. You can get them by:
+		</p>
+		<ul>
+			<li>compiling pathvisio.jar on your own by downloading <a href="http://pathvisio.org/">Pathvisio source code</a>
+			and putting pathvisio.jar in your CLASSPATH, or</li>
+			<li>downloading an old version of pathvisio.jar
+			<a href="http://bitbucket.org/leonth/reactome-wikipathways-exchange/raw/7d524ccb5c1c/lib/pathvisio.jar">here</a>
+			and putting it in your CLASSPATH.</li>
+		</ul>
 	</body>
 </html>
\ No newline at end of file
